server/src/db/migrations/20220131193845_addCharacters.cjs

```diff
@@ -0,0 +1,26 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+  return knex.schema.createTable("characters", (table) => {
+    table.bigIncrements("id")
+    table.string("name").notNullable()
+    table.string("gameTitle")
+    table.string("gameSeries").notNullable()
```

Comment:
Do we want `gameSeries` as not nullable or `gameTitle`? 
Previously on this PR, `gameTitle` was `.notNullable()`, but it seems like that was changed now

Reply:
Also, we want this to be inline with what we have in the model

Reply:
We have addressed the model.  This was a mix-up on our end.